DIAGRAMS-494: Added the network communicaton flow overview diagram (OSDOCS) (#107517)
openshift/openshift-docs: networking/networking_overview/understanding-networking.adoc

[role="_abstract"]
Understanding networking is essential for building resilient, secure, and scalable applications in {product-title}. From basic pod-to-pod communication to complex traffic routing and security rules, every component of your application relies on the network to function correctly.

> The following diagram shows the flow of external and internal network traffic among networking components for an {aws-first} external client that wants to connect to a pod that exists inside a cluster.

Reviewer: _wants_ is [anthropomorphism per ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism). 

Maybe:
```suggestion
The following diagram shows the flow of external and internal network traffic among networking components for an {aws-first} external client when connecting to a pod that exists inside a cluster.
```

```suggestion
The following diagram shows the flow of external and internal network traffic among networking components for an {aws-first} external client attempting to connect to a pod that exists inside a cluster.
```

Reviewer: Also, do you need _that exists inside_? 

```suggestion
The following diagram shows the flow of external and internal network traffic among networking components for an {aws-first} external client when connecting to a pod in a cluster.
```

Author: You sunk my battleship :ship: . I too have fallen into the anthro trap. Thanks! I'll update this line.
